Failed to load the ink package

Steps to reproduce:

1. start Atom

**Atom Version**: 1.0.4
**System**: Ubuntu 15.04
**Thrown From**: Atom Core


### Stack Trace

Failed to load the ink package

```
At Cannot find module './Loading'

Error: Cannot find module './Loading'
    at Module._resolveFilename (module.js:328:15)
    at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:270:25)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at Object.<anonymous> (/part_home/CSousa_Personal/Desktop/atom-packages/atom-ink/lib/ink.coffee:8:13)
    at Object.<anonymous> (/part_home/CSousa_Personal/Desktop/atom-packages/atom-ink/lib/ink.coffee:38:4)
    at Module._compile (module.js:452:26)
    at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
    at Module.load (module.js:347:32)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ],
    "disabledPackages": [
      "vim-mode-visual-block",
      "seti-icons",
      "wrap-guide",
      "keyboard-localization",
      "linter-julia",
      "link"
    ],
    "autoHideMenuBar": true,
    "followSymlinks": false
  },
  "editor": {
    "invisibles": {},
    "fontFamily": "Menlo",
    "scrollPastEnd": true,
    "scrollSensitivity": 10,
    "tabLength": 4,
    "fontSize": 12,
    "softWrap": true
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-clang, v0.8.7
hidpi, v0.7.0
indent-detective, v0.1.0
ink, v0.0.0
language-cmake, v0.1.4
language-cuda, v0.0.0
language-ini, v1.11.0
language-julia, v0.3.1
latex-completions, v0.3.0
linter, v1.3.1
linter-pep8, v1.0.1
minimap, v4.12.2
minimap-codeglance, v0.4.0
minimap-selection, v4.3.0
simple-drag-drop-text, v0.2.3

# Dev
No dev packages
```

--- a/lib/ink.coffee
+++ b/lib/ink.coffee
@@ -1,6 +1,6 @@
 {CompositeDisposable} = require 'atom'
 block = require './block'
-Loading = require './Loading'
+Loading = require './loading'
 Spinner = require './spinner'
 results = require './results'
 Console = require './console/console'

